apps/web/ee/lib/reminders/emailReminderManager.ts

```diff
@@ -0,0 +1,143 @@
+import { EventTypeAttendeeReminder } from "@prisma/client/";
+import client from "@sendgrid/client";
+import sgMail from "@sendgrid/mail";
+import dayjs from "dayjs";
+import isBetween from "dayjs/plugin/isBetween";
+import { v4 as uuidv4 } from "uuid";
+
+import { CalendarEvent } from "@lib/integrations/calendar/interfaces/Calendar";
+import prisma from "@lib/prisma";
+
+import reminderTemplate from "./templates/reminderEmailTemplate";
+
+const sendgridAPIKey = process.env.SENDGRID_API_KEY as string;
+const senderEmail = process.env.SENDGRID_EMAIL as string;
+
+sgMail.setApiKey(sendgridAPIKey);
+client.setApiKey(sendgridAPIKey);
+
+export const scheduleEmailReminder = async (
+  evt: CalendarEvent,
+  email: string,
+  attendeeReminder: EventTypeAttendeeReminder
+) => {
+  const { startTime } = evt;
+  const uid = evt.uid as string;
+  const currentDate = dayjs();
```

Comment:
Shouldn't we use UTC here?

Reply:
Hm. I think it is always best practice to standardise UTC to cover the possibilities of using the services without any issue when moving around different timezones. I'm not entirely sure yet if that would affect the functionality of the PR at all, but it's best to think over it.